src/ain.ts

```diff
@@ -165,27 +165,48 @@ export default class Ain {
   }
 
   /**
-   * Fetches the forger's address of a block with a block hash or block number.
-   * @param {string | number} blockHashOrBlockNumber The block hash or block number.
+   * Fetches the information of the given validator address.
+   * @param {string} address The validator address.
+   * @returns {Promise<any>}
+   */
+  getValidatorInfo(address: string): Promise<any> {
+    return this.provider.send('ain_getValidatorInfo', { address });
+  }
+
+  /**
+   * Fetches the validator list of a block with a block number.
+   * @param {number} blockNumber The block number.
+   * @returns {Promise<any>}
+   */
+  getValidatorsByNumber(blockNumber: number): Promise<any> {
+    return this.provider.send('ain_getValidatorsByNumber', { number: blockNumber });
+  }
+
+  /**
+   * Fetches the validator list of a block with a block hash.
+   * @param {string} blockHash The block hash.
+   * @returns {Promise<any>}
+   */
+  getValidatorsByHash(blockHash: string): Promise<any> {
+    return this.provider.send('ain_getValidatorsByHash', { hash: blockHash });
+  }
+
+  /**
+   * Fetches the forger's address of a block with a block number.
```

Comment:
question) what is `forger`?

Reply:
sometimes block proposer is called block forger or block producer. just replaced with 'block proposer' for readability. thanks!